addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -1,8 +1,6 @@
 .ribbon {
     width: 150px;
     height: 150px;
```

Comment:
Arbitrary values... why exactly `150px`?
Can't we, for example,  let the browser compute these dimensions based on the content?
If it's not possible can we use variables instead?

Reply:
We can't change those value dynamically without affecting the border of the ribbon, so I added a scss variable file